tests/app/tests/migrations/run_python_noop/0001_initial.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.0 on 2019-05-18 15:22
+
+from django.db import migrations
+from django.db.migrations import RunPython
+
+OtherRunPython = RunPython
+noop = OtherRunPython.noop
+
+
+def same_name(apps, schema_editor):
+    # original function
+    return
+
+
+def same_name_2(apps, schema_editor):
+    # original function 2
+    return
```

Comment:
I think you need to prove that you can handle any number of suffixes. To do that, I would make this test use "same_name_3" and "same_name_5". I think you need to prove that you can handle multiple similar suffixes, both consecutive ones (2 & 3) as well as gaps (3 & 5).

Do you also handle suffixes so that the same migration could also have "same_nameA", "same_name_A",  "same_nameB", "same_name_B", "same_name2"?

Reply:
I prove this in tests_utils.py::TestUtils.test_unique_function_names -- didn't see the need to do here also.